flexmeasures/data/tests/conftest.py

```diff
@@ -232,3 +231,349 @@ def setup_annotations(
+    values = [0] * 5 + list(range(7, 51, 7)) + list(range(50, 0, -7)) + [0] * 5
+    add_as_beliefs(db, testing_sensor2, values, time_slots, setup_sources["Seita"])
+    Solar2 = GenericAsset(
+        name="solar-2",
+        generic_asset_type=asset_type,
+    )
+    db.session.add(Solar2)
+    testing_sensor3 = Sensor(
+        name="solar2-production-price-sensor",
+        generic_asset=Solar2,
+        event_resolution=timedelta(hours=1),
+        unit="EUR/MWh",
+        knowledge_horizon=(at_date, {"knowledge_time": "2014-11-01T00:00+01:00"}),
+    )
+    db.session.add(testing_sensor3)
+    production_price = TimedBelief(
+        event_start="2015-01-01T00:00+01:00",
+        belief_time="2014-11-01T00:00+01:00",  # publication date
+        event_value=2.5,
+        source=setup_sources["Seita"],
+        sensor=testing_sensor3,
+    )
+    db.session.add(production_price)
+    testing_sensor4 = Sensor(
+        name="solar-power-2",
+        generic_asset=Solar2,
+        event_resolution=timedelta(hours=1),
+        unit="MW",
+        attributes={"capacity_in_mw": 2000},
+    )
+    db.session.add(testing_sensor4)
+    time_slots = initialize_index(
+        start=pd.Timestamp("2015-01-01").tz_localize("Europe/Amsterdam"),
+        end=pd.Timestamp("2015-01-02").tz_localize("Europe/Amsterdam"),
+        resolution="1H",
+    )
+    values = [0] * 5 + list(range(7, 51, 7)) + list(range(50, 0, -7)) + [0] * 5
+    add_as_beliefs(db, testing_sensor4, values, time_slots, setup_sources["Seita"])
+    Solar3 = GenericAsset(
+        name="solar-3",
+        generic_asset_type=asset_type,
+    )
+    db.session.add(Solar3)
+    testing_sensor5 = Sensor(
+        name="solar3-production-price-sensor",
+        generic_asset=Solar3,
+        event_resolution=timedelta(hours=1),
+        unit="EUR/MWh",
+        knowledge_horizon=(at_date, {"knowledge_time": "2014-11-01T00:00+01:00"}),
+    )
+    db.session.add(testing_sensor5)
+    production_price = TimedBelief(
+        event_start="2015-01-01T00:00+01:00",
+        belief_time="2014-11-01T00:00+01:00",  # publication date
+        event_value=3,
+        source=setup_sources["Seita"],
+        sensor=testing_sensor5,
+    )
+    db.session.add(production_price)
+    testing_sensor6 = Sensor(
+        name="solar-power-3",
+        generic_asset=Solar3,
+        event_resolution=timedelta(hours=1),
+        unit="MW",
+        attributes={"capacity_in_mw": 2000},
+    )
+    db.session.add(testing_sensor6)
+    time_slots = initialize_index(
+        start=pd.Timestamp("2015-01-01").tz_localize("Europe/Amsterdam"),
+        end=pd.Timestamp("2015-01-02").tz_localize("Europe/Amsterdam"),
+        resolution="1H",
+    )
+    values = [0] * 5 + list(range(7, 51, 7)) + list(range(50, 0, -7)) + [0] * 5
+    add_as_beliefs(
+        db, testing_sensor6, values, time_slots, setup_sources["Seita"]
+    )  # make sure that prices are assigned to price sensors
+    db.session.flush()
+    return {
```

Comment:
To simplify, I would return just the sensors:

```python
return testing_sensor1, testing_sensor2, ...
```

Then, when using the fixture, you can unwrap the return variables:

```python
testing_sensor1, testing_sensor2, ... = create_solar_plants
```